caas/kubernetes/provider/application/application.go

```diff
@@ -1416,7 +1416,7 @@ func (a *app) ApplicationPodSpec(config caas.ApplicationConfig) (*corev1.PodSpec
 		AutomountServiceAccountToken:  &automountToken,
 		ServiceAccountName:            a.serviceAccountName(),
 		ImagePullSecrets:              imagePullSecrets,
-		TerminationGracePeriodSeconds: pointer.Int64Ptr(300),
+		TerminationGracePeriodSeconds: pointer.Int64Ptr(10),
```

Comment:
Changed for quicker testing, revert later.

Reply:
Don't forget to revert before landing